jobs/intg-test-resources/latest/ansible-deployment/copy_deploy_script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o xtrace;
+
+INPUT_DIR=$2
+
+TEST_PROP_FILE=$INPUT_DIR/testplan-props.properties
+INFRA_PROP_FILE=$INPUT_DIR/infrastructure.properties
+GIT_USER=$(grep -m 1 -w "GIT_WUM_USERNAME" $TEST_PROP_FILE | cut -d'=' -f2)
+GIT_PASS=$(grep -m 1 -w "GIT_WUM_PASSWORD" ${TEST_PROP_FILE} | cut -d'=' -f2)
+keyFileLocation=$(grep -w "keyFileLocation" ${TEST_PROP_FILE} | cut -d'=' -f2)
+
+OS=$(grep  -w "OS" ${INFRA_PROP_FILE} | cut -d'=' -f2)
+WSO2InstanceName=$(grep -w "WSO2InstanceName" ${INFRA_PROP_FILE} | cut -d'=' -f2 | cut -d"/" -f3)
+
+ANSIBLE_REPO=$(grep -w "ANSIBLE_GIT_URL" $TEST_PROP_FILE | cut -d'=' -f2 | cut -d'/' -f3-)
+ANSIBLE_BRANCH=$(grep -w "ANSIBLE_GIT_BRANCH" $TEST_PROP_FILE | cut -d'=' -f2)
```

Comment:
Why cannot we derive this from the product version?

Reply:
Yes, we can. But I thought it's better to add as an input as in [1]. The branch name can be changed anytime. If so this is an easy way of updating scripts.   

[1] https://github.com/NishikaDeSilva/testgrid-job-configs/blob/test-nishi/Integration-Test/wso2am-2.1.0-intg-testgrid.yaml#L42